docs/BinSkimRules.md

```diff
@@ -890,17 +890,17 @@ Control-flow Enforcement Technology (CET) Shadow Stack is a computer processor f
 
 ### Description
 
-SourceLink information should be present in the PDB. This applies to binaries built with the C# and MSVC compilers. See https://aka.ms/sourcelink for more information.
+SourceLink information should be present in the PDB. This applies to binaries built with the C# and MSVC compilers. When enabled, SourceLink information is added to the PDB. That information includes the repository URLs and commit IDs for all source files fed to the compiler. The PDB should also be uploaded to a symbol server so that it can be discovered by a debugger such as Visual Studio. Developers can then step into the matching source code. Frictionless source-driven debugging provides a good user experience for consumers and also accelerates security response in the event of supply-chain compromise. See https://aka.ms/sourcelink for more information.
```

Comment:
>SourceLink [](http://example.com/codeflow?start=0&length=10)

@pharrington Updated the source link reporting a bit. Move explicative text to description, simplified the literal reporting a bit.

Reply:
You mentioned the wrong person, I am not involved in this.